api/database/migrations/2020_08_29_094025_change_account_id_to_string.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class ChangeAccountIdToString extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('social_accounts', function (Blueprint $table) {
+            $table->string('account_id')->change();
+        });
+    }
+
+    /**
+     * Reverse the migrations.
+     *
+     * @return void
+     */
+    public function down()
+    {
+        Schema::table('social_accounts', function (Blueprint $table) {
+            $table->dropColumn('account_id');
```

Comment:
The `down` method is for reverting database structure changes from the `up` method, but you are deleting column.